### PR TITLE
Cb 11 aws comprehend api

### PR DESCRIPTION
**Pull request by Nirvair-Sangha**
* **Please check if the PR fulfills these requirements**
- [x] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)

Adds new feature to allow the user to call the custom api to get the sentiment value.

* **What is the current behavior?What does the code do before you made the new changes?** 

Before the code uses gets to call on other APIs, but now it is able to call the Comprehend Custom API.

* **What is the new behavior?**

The code posts to /checkNews and analyzes for a response and returns a value depending on the response.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)

No

* **Other information**:

### CODE REVIEW

**Comment by Codubee** [comment]
Hey Nick, I think you might have pushed node_modules mistakenly.

File: backend/test/index_test.js [hunk 1]
```diff
@@ -13,4 +14,14 @@ describe('Tests get enpoint at /',()=>{
             done();
         })
     })
+
+    it('Should send paramters to /checkNews',(done)=>{
+        chai.request(app)
+        .post('/checkNews')
```
Comment (DHua5922) [inline backend/test/index_test.js:20]: Indent lines 20-25 by 1 tab so that the code is easier to read.

File: backend/index.js [hunk 2]
```diff
@@ -14,12 +11,30 @@ app.get('/',(req,res)=>{
...
 
+app.post('/checkNews', function (req, res) {
+    console.log(req.body);
+    
+    const body = {
+        news: req.body
+    }
+
+    axios.post('https://codubee-api.herokuapp.com/comprehension',body)
+    .then(function (response) {
```
Comment (DHua5922) [inline backend/index.js:22]: Indent lines 22-29 by 1 tab so that the code is easier to read.

File: backend/index.js [hunk 3]
```diff
@@ -14,12 +11,30 @@ app.get('/',(req,res)=>{
     res.send('Hello world');
 })
 
+app.post('/checkNews', function (req, res) {
+    console.log(req.body);
```
Comment (DHua5922) [inline backend/index.js:15]: Remove `console.log`. It is not needed in production.

**Review by DHua5922 — CHANGES_REQUESTED** [review]

**Comment by Nirvair-Sangha** [comment]
> Hey Nick, I think you might have pushed node_modules mistakenly.

Is there any way to revert this change, because I ended up not using the code from the body parser commit anyways.

**Comment by sunnyguan** [comment]
> > Hey Nick, I think you might have pushed node_modules mistakenly.
> 
> Is there any way to revert this change, because I ended up not using the code from the body parser commit anyways.

Should be fixed now, the `node_modules` were in the root directory, so I don't think they could've been used by either the frontend/backend code anyway.

**Review by sunnyguan — APPROVED** [review]
Looks good to me; tests worked both with Mocha as well as Postman.

**Review by Codubee — APPROVED** [review]

**Pull request merged**